extensions/markdown-language-features/src/preview/previewManager.ts

```diff
@@ -77,7 +77,7 @@ export class MarkdownPreviewManager extends Disposable implements vscode.Webview
 	private readonly _dynamicPreviews = this._register(new PreviewStore<DynamicMarkdownPreview>());
 	private readonly _staticPreviews = this._register(new PreviewStore<StaticMarkdownPreview>());
 
-	private _activePreview: IManagedMarkdownPreview | undefined = undefined;
+	public _activePreview: IManagedMarkdownPreview | undefined = undefined;
```

Comment:
Let's remove the `_` to make this into a proper public property. Even better through: create a new getter for `activeImage` that does not have a setter. That way external callers can't set the active preview

Reply:
We shouldn't need to make this property public anymore